bankingapp/bankingapp/settings.py

```diff
@@ -0,0 +1,137 @@
+"""
+Django settings for bankingapp project.
+
+Generated by 'django-admin startproject' using Django 3.0.3.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.0/ref/settings/
+"""
+
+import os
+from dotenv import load_dotenv
+
+
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+dot_env_path = os.path.join(BASE_DIR, '.env')
+load_dotenv(dot_env_path)
```

Comment:
Docker picks up the env. This isn't how you write take env for django. Look into environ and load envs into docker directly not django